src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -360,8 +371,31 @@ public void waitForSendReceiveData()
                     long download = bitrate.get("download");
                     long upload = bitrate.get("upload");
 
-                    if (download > 0 && upload > 0)
-                        return true;
+                    if (checkSend)
```

Comment:
can this be simplified to this ?
`return (!checkSend || upload > 0) && (!checkReceive || download > 0);`